wolfcrypt/src/port/caam/caam_driver.c

```diff
@@ -438,95 +438,93 @@ static void print_jdkek()
 /* instantiate RNG and create JDKEK, TDKEK, and TDSK key */
 static unsigned int wc_rng_start[] = {
     CAAM_HEAD | 0x00000006,
-    CAAM_OP | CAAM_CLASS1 | CAAM_RNG | 0x00000004, /* Instantiate RNG handle 0 with TRNG */
+    CAAM_OP | CAAM_CLASS1 | CAAM_RNG | 0x00000004, /* Instantiate RNG handle 0
+                                                      with TRNG */
     CAAM_JUMP | 0x02000001,  /* wait for Class1 RNG and jump to next cmd */
     CAAM_LOAD | 0x00880004,  /* Load to clear written register */
     0x00000001, /* reset done interrupt */
     CAAM_OP | CAAM_CLASS1 | CAAM_RNG | 0x00001000   /* Generate secure keys */
 };
 
 
-
 /* Initialize CAAM RNG
  * returns 0 on success */
 int caamInitRng(struct CAAM_DEVICE* dev);
 int caamInitRng(struct CAAM_DEVICE* dev)
 {
     DESCSTRUCT desc;
-    unsigned int reg, status;
-    int ret = 0;
+    unsigned int reg, entropy_delay;
+    int ret = 0, i;
 
+    /* set up the job description for RNG initialization */
     memset(&desc, 0, sizeof(DESCSTRUCT));
+    desc.desc[desc.idx++] = CAAM_HEAD; /* later will put size to header*/
+    for (i = 1; i < 6; i = i + 1) {
```

Comment:
Magic numbers.
Use a define for 6.
i++?

Reply:
Put RNG_START_SIZE into array definition or base define off of sizeof.
And i = i + 1 => i++